sources/packages/backend/apps/workers/src/services/disbursement-schedule/disbursement-schedule.service.ts

```diff
@@ -82,6 +82,38 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
     await this.repo.save(disbursements);
   }
 
+  /**
+   * Update the has_estimated_awards property to false for any disbursement schedule
+   * for the assessment id if the final award total is $0.
+   * @param assessmentId assessment id.
+   * @param finalAwardTotal final award total from the assessment data.
+   */
+  async updateDisbursementsHasEstimatedAwards(
```

Comment:
This service wouldn't be required if we follow the path as per https://github.com/bcgov/SIMS/pull/4010/files#r1863853214